src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstSide, int secondSide, int height, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+        super.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstSide + secondSide) * height / 2);

```

Comment:
Use parentheses to make the formula more readable and ensure the correct order of operations. It should be `0.5 * (firstSide + secondSide) * height`.